src/commands/index.js

```diff
@@ -1,27 +1,27 @@
 module.exports = {
```

Comment:
wonder if it's better to do this dynamically:

```js
module.exports = sortObjectKeys(module.exports); // see below
```

Since this is done in other places perhaps a utility function:

```js
const sortObjectKeys = (object) => (
  Object.keys(object).sort().reduce((sorted, key) => (sorted[key] = object[key], sorted), {})
);
```